docs/network-flow-visibility.md

```diff
@@ -43,6 +47,51 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+### Prerequisites
+
+To get started, make sure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following `antrea-agent` config parameter in
```

Comment:
perhaps this paragraph can be simplified as follows:

To install Theia, ensure the Flow Exporter feature must be enabled in Antrea deployment manifest, in the following way:

Reply:
Thanks, updated! I remove the "must" as it seem a little bit duplicated with "ensure".

Reply:
you're right, thanks!